exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/FunctionProvider.kt

```diff
@@ -438,6 +439,60 @@ abstract class FunctionProvider {
         return "INSERT INTO ${transaction.identity(table)} $columnsExpr $valuesExpr"
     }
 
+    open fun merge(
```

Comment:
Missing KDocs

Reply:
Updated also for mergeSelect() in the same file